code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -1357,20 +1470,35 @@ func (sc *StorageSmartContract) cancelAllocationRequest(
 		sps = append(sps, sp)
 	}
 
-	err = sc.finishAllocation(t, alloc, passRates, sps, balances, conf)
+	isEnterprise := false
+	_ = chainstate.WithActivation(balances, "electra", func() error {
```

Comment:
Should not ignore the error

Reply:
What error can we return here? We are returning nil all the time so ignored the check.